app/components/homepage-components/ServicesCard.tsx

```diff
@@ -1,37 +1,57 @@
 "use client";
 
-import { useState } from "react";
-import Image from "next/image";
+import { motion } from "framer-motion";
 import { useTheme } from "next-themes";
+import { useState, useEffect } from "react";
+import Image from "next/image";
 
 import { ServiceProvidedType } from "@/types";
 interface ServicesCardProps {
   service: ServiceProvidedType;
+  delay?: number;
 }
 
-const ServicesCard = ({ service }: ServicesCardProps) => {
+const ServicesCard = ({ service, delay }: ServicesCardProps) => {
   const { theme } = useTheme();
   const [isMouseOver, setIsMouseOver] = useState<boolean>(false);
+  const [backgroundColor, setBackgroundColor] = useState("");
+  const [hoverBackgroundColour, setHoverBackgroundColour] = useState("");
 
-  let imageSrc;
+  const imageSrc = isMouseOver
+    ? theme === "light"
+      ? service.imageHoverLight
+      : service.imageHoverDark
+    : service.imageSrc;
 
-  if (isMouseOver) {
+  useEffect(() => {
     if (theme === "light") {
-      imageSrc = service.imageHoverLight;
+      setHoverBackgroundColour("#0252CD");
+      setBackgroundColor("#F3F8FF");
     } else {
-      imageSrc = service.imageHoverDark;
+      setHoverBackgroundColour("#428DFF");
+      setBackgroundColor("#192333");
     }
-  } else {
-    imageSrc = service.imageSrc;
-  }
+  }, [theme]);
 
   return (
-    <div
-      className={`w-full rounded-xl px-6 py-7 shadow-lg md:h-[18rem] md:w-[19rem] xl:w-full ${
-        isMouseOver
-          ? "bg-primaryLight dark:bg-primaryDark"
-          : "bg-white800 dark:bg-black300"
-      }`}
+    <motion.div
+      initial={{
+        x: "25%",
+        opacity: 0,
+      }}
+      animate={{ x: 0, opacity: 1, backgroundColor }}
+      whileHover={{
+        y: "10%",
+        transition: { duration: 0.2 },
+        backgroundColor: hoverBackgroundColour,
+        boxShadow: "30px 30px 40px #0252cd50",
```

Comment:
Take a look at the box shadow and ensure you're getting what you expected. I don't see any colors under "#0252cd50." If this is a mistake, it could have been prevented if you correctly defined your colors in the tailwind config...

Reply:
@cdsalty 

Managed to fix this one